app/client/src/components/GenerateMicroreactURL.vue

```diff
@@ -11,7 +11,7 @@
 
       <button
         v-if='tokenAvailable'
-        @click='buildMicroreactURL(cluster)'
+        @click='buildMicroreactURL({cluster, token: microreactToken})'
```

Comment:
Not sure how this change is related to the filename mapping?

Reply:
It is not.. Just realised that the behaviour of generating the microreactURL was sometimes not as expected and found this little bug, where I had forgotten to provide all parameters to the action. Thought this fix is so small I can just add that with this PR rather than opening a new PR just for this one line of code?